### PR TITLE
31 create database

### PR DESCRIPTION
**Pull request by parslie**
Note that we haven't rebased the branch yet, so this should be done before merging.

### CODE REVIEW

**Comment by parslie** [comment]
Note that we haven't rebased the branch yet, so this should be done before merging.

File: src/database/runner.py
Comment (edvgi966) [inline src/database/runner.py:1]: missing docstrings of functions

File: src/database/server.py
Comment (edvgi966) [inline src/database/server.py:1]: missing docstrings

**Review by edvgi966 — CHANGES_REQUESTED** [review]
Code looks good but many larger and important functions are not properly commented.

**Review by grufe — CHANGES_REQUESTED** [review]
Add directory ./src/.idea/ to gitignore.

**Pull request closed without merging**